install-instructions.md

```diff
@@ -120,17 +120,73 @@ For macOS or Linux [you need Java](https://adoptium.net/) to run the Fabric Inst
+
+* This process has a lot of steps which you must follow carefully.
+* The tutorial is made for Android devices - phones and tablets. iOS devices _could_ work, but that requires jailbreak or sideloading an alternative store, the steps are probably more complicated.
+* Your device must be "mid-range" or "high-end" if you want decent FPS. 6+ GB RAM recommended.
+* Pojav Launcher supports Android 5 and up, but these instructions were tested on Android 13.
+* Your device will likely heat and consume a lot of battery as you play. It is _not_ as device-optimized as Bedrock Edition.
+* Attempts have been made to improve performance, but depending on your device you may need to adjust video settings.
+* In the worst case, you may not get it running at all. Try again after Pojav Launcher updates.
+
+<details><summary><a href="https://play.google.com/store/apps/details?id=com.mojang.minecraftpe">I don't agree and just want to play Minecraft</a></summary></details>
+
+<details>
+  <summary>I agree and would like to see the instructions</summary>
+
+1. [Download Pojav Launcher](https://play.google.com/store/apps/details?id=net.kdt.pojavlaunch)
+   * Do not fall into scam websites and apps. The only official site for it is https://github.com/PojavLauncherTeam/PojavLauncher
+2. [Download Files](https://play.google.com/store/apps/details?id=com.marc.files)
+   * This is a shortcut to your system's file manager. Other file managers may need root to follow these instructions.
+3. [Download Fabric installer](https://fabricmc.net/use/installer/) (Universal/.JAR)
+4. [Download the latest **MultiMC version** of Fabulously Optimized](https://www.curseforge.com/minecraft/modpacks/fabulously-optimized/files)
+   * On CurseForge's new UI, `⌵` → `Download file`
+   * Don't see the MultiMC version? Click the title of the CurseForge version you need and scroll down, you'll find the MultiMC variant there.
+      * Alternatively, look for it in the sidebar, under "server packs". Don't worry, these are not server packs, just marked as such for findability.
+   * If you don't see any mods inside the zip or only see one, you downloaded the wrong version.
+5. Download Fabulously Optimized patches for Pojav Launcher: [1.19.3](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/Fabulously-Optimized/fabulously-optimized/tree/pojav/PojavLauncher/1.19.3)
+   * This optimizes the modpack for mobile users.
+6. Run Pojav Launcher
+7. Tap `Add account` → `Microsoft account` and sign in
+8. Tap `(Default) - Unknown` → `Create new profile` → `SAVE`
+9. Tap `Play`. Wait for it to fully load to the title screen, then exit the game.
+    * If you get a RAM warning ("The current amount of..."), tap `OK` for now. It usually means you just need to close background apps for better performance.
+    * If you get a black screen: quit the game, go back to Pojav → `🎛️` → `Video and renderer` → `Renderer` → (choose any) → ⬅️ → ⬅️ → try playing again
```

Comment:
IMS also said to use Zink when available as this provides actually better performance and to not disable Sodium's chunk rendering via patches.